Kickstarter-iOS/Views/Cells/FindFriendsCell.swift

```diff
@@ -13,8 +13,10 @@ final class FindFriendsCell: UITableViewCell, ValueCell, NibLoading {
   private let viewModel: FindFriendsCellViewModelType = FindFriendsCellViewModel()
 
   func configureWith(value: SettingsCellValue) {
-    let user = value.user
-    self.viewModel.inputs.configure(with: user)
+    _ = self
```

Comment:
Are the following changes necessary?

* `FindFriendsCell`
* `SettingsAccountWarningCell`
* `SettingsTableViewCell`
* `SettingsViewController`
* some of the `Settings.storyboard` (mostly `UILabel`/`UITextField` prop names)
* `HelpType`
* `SettingsCellType`
* `README.md` (just the installation instrucitons) necessary?

They appear as duplicates from previous PRs so we should only include changes related to change email/password input fields a11y.